src/main/webapp/common_util.js

```diff
@@ -13,16 +13,22 @@
 // limitations under the License.
 
 /*
- * Extracts video id from full url
- */ 
-function cleanseUrl(url) {
-  // Split web address from parameters, extract first parameter
-  // TODO: Add checks to make this work if video is not first parameter.
-  var videoId = url.split("?");
-  videoId = (videoId.length > 1) ? videoId[1].split("&")[0] : videoId[0];
-
-  // If param name present, remove it to isolate video Id.
-  videoId = videoId.replace("v=", "");
-  
+ * @param url: url of the video to be analyzed
+ * @returns: just the video id of that video
+ * Credit for RegEx method derived from top answer on: 
+ * https://stackoverflow.com/questions/28735459/how-to-validate-youtube-url-in-client-side-in-text-box
+ */	 
+function extractYouTubeUrl(url) {
+  var videoId = '';
```

Comment:
Should we still call it cleanseURL? As of right now the other js files are passing in a URL. Do we need a video id or a youtube url to be passed into the servlet?

Reply:
Maybe we could call this extractVideoID instead

Reply:
Hmm okay, I think that is why I changed it from cleanseUrl as the current name is more descriptive.